editor/src/clj/editor/app_view.clj

```diff
@@ -2143,7 +2143,7 @@ If you do not specifically require different script states, consider changing th
                            (fn [successful?]
                              (when successful?
                                (if (some-> output-directory .isDirectory)
-                                 (ui/open-file output-directory)
+                                 (when (prefs/get-prefs prefs "open-bundle-target-folder" false) (ui/open-file output-directory))
```

Comment:
The `false` argument here is being used when there is no preference set. It should be `true` here as well since the default is `true` in `prefs_dialog.clj` below.

Otherwise, users who haven't touched the preference will see it as checked, but it will behave as if it is unchecked.

Reply:
Got it, thx.
Fixed.